src/nmi/render_mode_play_and_demo.asm

```diff
@@ -93,7 +93,7 @@ render_mode_play_and_demo:
         and #RENDER_SCORE
         beq @renderHz
 
-        ; 8 safe tile writes freed from stats / hz
+        ; 9 safe tile writes freed from stats / hz
```

Comment:
I think this should be 7? it's talking about writes free from what the original was doing

although it doesnt really matter what the delta is to the original at this point, maybe we can rewrite it differently. I'll have a think